build.js

```diff
@@ -200,6 +201,10 @@ return Promise.resolve()
     .then(() => _relativeCopy('README.md', rootFolder, distFolder))
     .then(() => console.log('Package files copy succeeded.'))
   )
+  // Global Styles
+  .then(() => _preprocessStyle(path.join(srcFolder, 'styles'), 'base.less', path.join(distFolder, 'styles.css'))
```

Comment:
We're just supporting less on global styles right now, but I have a feeling that we'll want to convert the component styles as well